darkmoon/api/v1/metadata/views.py

```diff
@@ -122,7 +112,7 @@ async def upload_metadata(
     file: Metadata,
     collection: AsyncIOMotorCollection = Depends(get_file_metadata_collection),
 ) -> None:
-    """Fast API POST function for incoming files.
+    """Fast API post function for incoming files.
```

Comment:
"Adds metadata from files to the database"
Saying that this is a FastAPI POST function is redundant.